### ISSUE TITLE
How can I configure VS Code to recognize the AWS functions in the template.yaml file?

### ISSUE THREAD

**Issue by macfarmw** [aws/aws-toolkit-vscode#1891]
**Desktop**
-   OS: Windows 10
-   VS Code version: 1:58.2
-   AWS Toolkit extension version: 1.27.0

**Question**
How can I configure VS Code to recognize the AWS functions in the template.yaml file?

**Steps**
1. Installed the AWS Toolkit and prerequisites.
2. Added AWS Toolkit, YAML, and C# extensions to VS code.
3. Ran: AWS: Create Lambda SAM Application and selected the Hello World sample.
4. Opened the template.yaml file from the Explorer

**Results**
The lines with !Sub and !GetAtt show as syntax errors and the message is unknown tag.

What other steps can I do so that the AWS functions are recognized as valid?

**Comment by JadenSimon** [comment]
Hey @macfarmw ,

We just merged a PR (#1748) that should add CFN tags automatically. Until then, you can get rid of the syntax errors by adding the following to your VS Code `settings.json` file:
```json
    "yaml.customTags": [
        "!And",
        "!And sequence",
        "!If",
        "!If sequence",
        "!Not",
        "!Not sequence",
        "!Equals",
        "!Equals sequence",
        "!Or",
        "!Or sequence",
        "!FindInMap",
        "!FindInMap sequence",
        "!Base64",
        "!Join",
        "!Join sequence",
        "!Cidr",
        "!Ref",
        "!Sub",
        "!Sub sequence",
        "!GetAtt",
        "!GetAZs",
        "!ImportValue",
        "!ImportValue sequence",
        "!Select",
        "!Select sequence",
        "!Split",
        "!Split sequence"
    ]
```
The settings file can be accessed via the command `Preferences: Open Workspace Settings (JSON)` or alternatively found in `.vscode` in your current workspace. You can also add this to your user settings if you want it applied to all workspaces.

**Comment by macfarmw** [comment]
Well, that was easy! Thanks.
